flask-call-tracking_call-control/.env.sample

```diff
@@ -0,0 +1,8 @@
+TELNYX_API_KEY="YOUR_API_KEY"
+TELNYX_CONNECTION_ID="YOUR_CONNECTION_ID"
+MESSAGING_PROFILE_ID="YOUR_MESSAGING_PROFILE_ID"
+
+DB_SERVER_NAME="localhost"
+DB_USERNAME="root"
+DB_PASSWORD=""
+DB_NAME="cctracker"
```

Comment:
Update these to match the code

```
mysql_db = MySQLDatabase(os.getenv('DATABASE_NAME'),
                         user=os.getenv('DATABASE_USER'),
                         password=os.getenv('DATABASE_PASSWORD'),
                         host=os.getenv('DATABASE_HOST'),
                         port=3306)
```

Or update the code to match these in `database.py`